clang/include/clang/Driver/Options.td

```diff
@@ -1862,6 +1862,9 @@ def fsycl_device_code_split_EQ : Joined<["-"], "fsycl-device-code-split=">,
 def fsycl_device_code_split : Flag<["-"], "fsycl-device-code-split">, Alias<fsycl_device_code_split_EQ>,
   AliasArgs<["per_source"]>, Flags<[CC1Option, CoreOption]>,
   HelpText<"Perform SYCL device code split in the per_source mode i.e. create a device code module for each source (translation unit)">;
+defm sycl_id_queries_fit_in_int : OptOutFFlag<"sycl-id-queries-fit-in-int",
```

Comment:
Just curious.
What's the difference between `def` and `defm`? I can't find proper documentation for it.

Reply:
`defm` I believe is to use a multiclass to define the option, as `OptInFFlag` and `OptOutFFlag` are used here instead of the regular definitions.   `OptInFFlag` and `OptOutFFlag` do additional expansions creating 2 entries.